libs/openFrameworks/utils/ofThreadChannel.h

```diff
@@ -39,16 +84,44 @@ class ofThreadChannel{
 		}
 	}
 
-	/// receives a new value in the passed parameter
-	/// and returns true or returns false if there
-	/// is no data available or the channel was closed
-	bool tryReceive(T & ret){
+	/// \brief If available, receive a new sent value without blocking.
+	///
+	/// ofThreadChannel::tryReceive is similar to ofThreadChannel::receive,
+	/// except that it will block the receiving thread as it waits for sent
+	/// data.  If no data is available, it will return false and continue
+	/// immediately.
+	///
```

Comment:
tryReceive won't block the receiving thread and instead returns immediately if there's no data.